### PR TITLE
Update ODS-469 to use Notebook CR for last spawn details

### PR DESCRIPTION
**Pull request by lugi0**
Signed-off-by: Luca Giorgi <lgiorgi@redhat.com>

### CODE REVIEW

File: tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot [hunk 1]
```diff
@@ -525,11 +525,22 @@ Get List Of All Available Container Size
     [Return]    ${size}
 
 Get Previously Selected Notebook Image Details
+    [Documentation]  Returns image:tag information from the Notebook CR for a user
+    ...    or minimal-gpu:default if the CR doesn't exist
```
Comment (jgarciao) [inline tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot:529]: This kw is currently customized for one test case. Could you add a parameter with the username? Or move the kw to the test suite?

Comment (jgarciao) [inline tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot:529]: ```...    or minimal-gpu:default, if the CR doesn't exist (default pre selected image in spawner)```

File: tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot [hunk 2]
```diff
@@ -525,11 +525,22 @@ Get List Of All Available Container Size
...
     ${safe_username} =   Get Safe Username    ${TEST_USER.USERNAME}
-    ${user_name} =    Set Variable    jupyterhub-singleuser-profile-${safe_username}
-    ${user_configmap} =    Oc Get    kind=ConfigMap    namespace=redhat-ods-applications
-    ...    field_selector=metadata.name=${user_name}
-    @{user_data} =    Split String    ${user_configmap[0]['data']['profile']}    \n
+    ${user_name} =    Set Variable    jupyter-nb-${safe_username}
+    # The TC using this kw only cares about the image:tag information, let's get that
```
Comment (jgarciao) [inline tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot:532]: Could you remove commented code?

**Review by jgarciao — CHANGES_REQUESTED** [review]

File: tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot (hunk 1 above)
Comment (lugi0) [reply tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot]: moved to test suite

**Review by jgarciao — APPROVED** [review]
lgtm

**Review by bdattoma — APPROVED** [review]

**Pull request merged**